lnclient/lnd/wrapper/lnd.go

```diff
@@ -46,11 +46,16 @@ func NewLNDclient(lndOptions LNDoptions) (result *LNDWrapper, err error) {
 		if err != nil {
 			return nil, err
 		}
-		cp.AppendCertsFromPEM(cert)
-		creds = credentials.NewClientTLSFromCert(cp, "")
+		if !cp.AppendCertsFromPEM(cert) {
+			return nil, errors.New("failed to append certificate")
+		}
+		creds = credentials.NewTLS(&tls.Config{
+			ClientCAs:          cp,
+			InsecureSkipVerify: true,
```

Comment:
I think we should verify the cert if it's provided. But everything else looks good!